modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/TabsGuide.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayTabs from '@clayui/tabs';
+import React, {useState} from 'react';
+
+import TokenGroup from '../components/TokenGroup';
+import TokenItem from '../components/TokenItem';
+
+const tertiaryTab = (
```

Comment:
Might as well format this like a constant: TERTIARY_TAB

Reply:
Fixed